testdata/project-v3-with-deploy-image/config/rbac/role.yaml

```diff
@@ -51,6 +51,13 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - example.com.testproject.org
```

Comment:
The RBAC is wrong here. 
The events are not an API from the project we need the api from k8s that is responsible for.
The group/domain in the marker is wrong.

Reply:
the API group for events should be core, right?

Reply:
the API group for events should be core, right?